projects/pgai/pgai/cli.py

```diff
@@ -12,12 +12,12 @@
 import psycopg
 import structlog
 from ddtrace import tracer
-from dotenv import load_dotenv
+from dotenv import load_dotenv, find_dotenv
 from pytimeparse import parse  # type: ignore
 
 from .__init__ import __version__
 
-load_dotenv()
+load_dotenv(dotenv_path=find_dotenv(usecwd=True))
```

Comment:
Should we also look at the user's home directory as well for any dotenv files? Could make it easier to use pgai from anywhere on your computer, vs having to configure it per specific location.

Reply:
Correct me if I'm wrong @MasterOdin but that will force env vars to be uniquely named (e.g. prefixed always with PGAI_) in order to avoid setting env vars for other applications in the home dir that could be taken by pgai.

Reply:
yeah let's not be too clever. This will already work if we are calling the program in a subdir of the home dir. I don't think we need any extra logic